File: Py/Classi/Scuola/persona.py
class Persona:

    # Costruttore
    def __init__(self, nome, cognome, matricola):
        self.nome = nome
        self.cognome = cognome
        self.matricola = matricola
    
    # Getters
    def getNome(self):
        return self.nome
    
    def getCognome(self):
        return self.cognome
    
    def getMatricola(self):
        return self.matricola
    
    # Setters
    def setNome(self, nome):
        self.nome = nome

    def setCognome(self, cognome):
        self.cognome = cognome

    def setMatricola(self, matricola):
        self.matricola = matricola

File: Java/Scuola/src/Persona.java
public class Persona {

    // Variables Assignament
    private String name;
    private String surname;
    private String matricola;

    // Constructor
    public Persona(String nameString, String surnameString, String matricolaString) {
        this.name = nameString;
        this.surname = surnameString;
        this.matricola = matricolaString;
    }

    // Getters
    public String getName() {
        return this.name;
    }

    public String getSurname() {
        return this.surname;
    }

    public String getMatricola() {
        return this.matricola;
    }

    // Setters
    public void setName(String nameString) {
        this.name = nameString;
    }

    public void setSurname(String surnameString) {
        this.surname = surnameString;
    }

    public void setMatricola(String matricolaString) {
        this.matricola = matricolaString;
    }
}